eng/Version.Details.xml

```diff
@@ -95,6 +95,10 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>7f2b222f92d323b7ca522a06c7116f1ec25d12ca</Sha>
     </Dependency>
+    <Dependency Name="System.Formats.Nrbf" Version="9.0.0-preview.7.24401.8" CoherentParentDependency="Microsoft.Private.Winforms">
```

Comment:
Are we going to add reference for this in versions.props as well?

Reply:
Version.props already has the reference to this package. We missed adding it here.